Obleak.Fody/ModuleWeaver.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Mono.Cecil;
+
+namespace Obleak.Fody
+{
+    public class ModuleWeaver
```

Comment:
Does this type do something magical which isn't apparent from its lack of usage elsewhere in the code?